pkg/convert/arb_test.go

```diff
@@ -92,6 +94,58 @@ func Test_FromArb(t *testing.T) {
 					`),
 			wantErr: errors.New("'Description' expected type 'string', got unconvertible type 'map[string]interface {}'"),
 		},
+		{
+			name: "With locale",
+			data: []byte(`
+      {
+        "@@locale": "en",
+        "title": "Hello World!",
+        "@title": {
+          "description": "Message to greet the World"
+        }
+      }
+			`),
```

Comment:
?

```suggestion
      }`),
```

Reply:
I think this
<img width="608" alt="image" src="https://user-images.githubusercontent.com/69954124/223404657-ff97eeee-abd9-4418-837a-967180e096fb.png">
looks cleaner than this
<img width="627" alt="image" src="https://user-images.githubusercontent.com/69954124/223404729-f2e64928-1cf5-4698-9eed-90611134246c.png">

GitHub makes different indentation than vscode